server/cmd/start.go

```diff
@@ -42,9 +42,6 @@ func Start(insecure bool, port int, namespaced bool, managedNamespace string) {
 	router.Use(gin.Logger())
 	router.RedirectTrailingSlash = true
 	router.Use(static.Serve("/", static.LocalFile("./ui/build", true)))
-	if namespaced {
```

Comment:
With this, when querying a namespace which is not allowed, it always shows the pipelines of the installation namespace, which is confusing on the UI, and breaks the following up features.